tutorials/misc/plot_ecog.py

```diff
@@ -183,20 +166,118 @@ def animate(i, activity):
                                frames=show_power.shape[1],
                                interval=100, blit=True)
 
+###############################################################################
+# Now let's animate the raw time series data and add events related
+# to seizure onset that are marked in the annotations. This will be done
+# very similarly to the high gamma visualization.
+#
+# As can be seen in the plot below, the seizure originates in the temporal
+# lobe and spreads to new electrodes when electrode label names come up in
+# the title. The seizure eventually becomes generalized and then subsides.
+
+# colormap to view spectral power - RdBu_r works well for directional
+# data such as voltage time series, where positive and negative values
+# matter and there is a true 0 point
+cmap = 'RdBu_r'
+
+# Apply a notch filter to remove 60 Hz line noise and harmonics
+raw_notched = raw.copy().pick_types(
+    meg=False, ecog=True).notch_filter([60, 120])
```

Comment:
after @larsoner's addition of downsampling, this would need to change (otherwise second notch is above Nyquist), but it looks like he added a notch filter up above anyway so we shouldn't need it here at all

```suggestion
    meg=False, ecog=True)
```